views/static-pages/archived.njk

```diff
@@ -0,0 +1,27 @@
+{% from "components/hero.njk" import hero %}
+{% from "components/content-box/macro.njk" import contentBox %}
+
+{% extends "layouts/main.njk" %}
+
+{% block extraHead %}
+    <meta http-equiv="refresh" content="2;url={{ archiveUrl }}" />
```

Comment:
Forgot this tag existed. Would you mind replacing the one in `user/interstitial.njk` with this same technique (and make it 2 seconds rather than 3 there)?